### PR TITLE
LPS-80617 Make sure the current configuration is marked as active in the menu

### PR DESCRIPTION
**Pull request by JorgeFerrer**
/cc @pei-jung

### CODE REVIEW

**Comment by JorgeFerrer** [comment]
ci:test:sf

**Comment by JorgeFerrer** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/mentions/mentions-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -2,4 +2,4 @@ asset-entry-mention-email-body=Asset Entry Mention Email Body
 asset-entry-mention-email-subject=Asset Entry Mention Email Subject
 comment-mention-email-body=Comment Mention Email Body
 comment-mention-email-subject=Comment Mention Email Subject
-mentions-group-service-configuration-name=Mentions Service
\ No newline at end of file
+mentions-group-service-configuration-name=Mentions
```
Comment (drewbrokke) [inline modules/apps/mentions/mentions-api/src/main/resources/content/Language.properties]: Don't we want this to be `Service` instead?  Like we're doing to Knowledge Base above?

**Comment by drewbrokke** [comment]
Pull request forwarded to https://github.com/brianchandotcom/liferay-portal/pull/59470. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**